init.lua

```diff
@@ -366,6 +354,7 @@ local on_digiline_receive_string = function(pos, node, channel, msg)
 				local skip = tonumber(string.sub(msg, 12))
 				led_marquee.scroll_text(pos, nil, skip)
 			elseif msg == "get" then -- get the master panel's displayed char as ASCII numerical value
+				-- luacheck: ignore 631
 				digilines.receptor_send(pos, digiline.rules.default, channel, tonumber(string.match(minetest.get_node(pos).name,"led_marquee:char_(.+)"))) -- wonderfully horrible string manipulaiton
```

Comment:
This should fix it. (I don't think the comment is necessary)
```suggestion
				digilines.receptor_send(pos, digiline.rules.default, channel, tonumber(string.match(node.name, "led_marquee:char_(.+)"))) 
```